django_filters/filterset.py

```diff
@@ -50,13 +51,19 @@ def all_valid(fieldset):
             count = len([i for i in fieldset if cleaned_data.get(i)])
             return 0 < count < len(fieldset)
 
+        def is_ordering_invalid(fieldset):
+            if ordering_field is None or not form.cleaned_data.get(ordering_field):
+                return
+            count = len([i for i in fieldset if i in form.cleaned_data[ordering_field]])
+            return 0 < count < len(fieldset)
```

Comment:
The intent is to fail if any field in the fieldset is not present in the ordering field's data. Checking count bounds does not clearly convey this intent. Maybe something like:

```python
ordering_data = form.cleaned_data[ordering_field]

# tests for presence in ordering data, returns list of true/false
together_ordering = [field in ordering_data for field in fieldset]

if any(together_results):
    return all(together_results)
return True  # default to success
```